server/app/models.py

```diff
@@ -733,7 +744,6 @@ class Score(Base):
     score = ndb.IntegerProperty()
     message = ndb.TextProperty() # Plain text
     grader = ndb.KeyProperty(User) # For autograders, the user who authenticated
-    server_time = ndb.DateTimeProperty(auto_now_add=True)
 
```

Comment:
@Sumukh @LuiseV Did one of you guys add this line? I've been reapplying old commits and am not sure if this line should be deleted.